portal-web/test/functional/com/liferay/portalweb/tests/mytests/MyDMTests.testcase

```diff
@@ -0,0 +1,56 @@
+@component-name = "portal-document-management"
+definition {
+
+	property testray.main.component.name = "Document Management";
+
+	setUp {
+		TestCase.setUpPortalInstance();
+
+		User.firstLoginPG();
+
+		Navigator.openURL();
+
+		task ("Given the DM Portlet is added to the page") {
+			JSONLayout.addPublicLayout(
+				groupName = "Guest",
+				layoutName = "DM Page");
+
+			JSONLayout.addWidgetToPublicLayout(
+				column = 1,
+				groupName = "Guest",
+				layoutName = "DM Page",
+				widgetName = "Documents and Media");
+
+			Navigator.gotoPage(pageName = "DM Page");
+
+			DMDocument.enableActionsMenuOnPortlet();
+
+			LexiconEntry.changeDisplayStyle(displayStyle = "list");
+		}
```

Comment:
Hey, could you explain to me how using this macro affects your test?

Reply:
I noticed that most of the DM tests that used the page widget put that macro in the setUp block. I believe the other macros work with that display style, so I put that macro in my test as well.

Reply:
It worked, but since I'll refactor the code so the DM can be created through the Product Menu, I'll delete this line. Thank you! :+1: